src/ui/view/home.rs

```diff
@@ -241,222 +245,262 @@ impl StakeholderOverview {
+                            .push(text::simple(" vaults")),
+                    )
+                    .width(Length::Fill)
+                    .align_x(Align::End),
+                ),
+        ).push(Container::new(Row::new().push(text::small(" "))));
+    card::white(Container::new(col.spacing(20)))
 }
 
-fn acked_funds_overview_card<'a, T: 'a>(
+fn secured_funds_overview_card<'a, T: 'a>(
     ctx: &Context,
-    nb_secured_vaults: u64,
-    secured_amount: u64,
+    secure: Option<&(u64, u64)>,
     securing: Option<&(u64, u64)>,
+    activating: Option<&(u64, u64)>,
 ) -> Container<'a, T> {
-    let mut col = Column::new().push(
-        Row::new()
-            .push(
-                Container::new(
-                    Row::new()
-                        .push(shield_check_icon())
-                        .push(text::simple("  Acknowledged funds:"))
-                        .align_items(Align::Center),
-                )
-                .width(Length::Fill),
-            )
-            .push(
-                text::bold(text::simple(
-                    &ctx.converter.converts(secured_amount).to_string(),
-                ))
-                .size(20),
-            )
-            .push(text::simple(&format!(" {},   ", ctx.converter.unit)))
-            .push(text::bold(text::simple(&nb_secured_vaults.to_string())))
-            .push(text::simple(" vaults"))
-            .align_items(Align::End),
-    );
-
-    if let Some((nb_securing_vaults, securing_amount)) = securing {
-        col = col.push(separation().width(Length::Fill)).push(
+    let (nb_secured_vaults, secured_amount) = secure.unwrap_or(&(0, 0));
+    let (nb_activating_vaults, activating_amount) = activating.unwrap_or(&(0, 0));
+    let mut col = Column::new()
+        .push(
             Row::new()
                 .push(
                     Container::new(
                         Row::new()
-                            .push(text::small(
-                                "funds waiting other stakeholder acknowledgment:",
-                            ))
+                            .push(shield_check_icon())
+                            .push(text::bold(text::simple("  Secured funds")))
                             .align_items(Align::Center),
                     )
                     .width(Length::Fill),
                 )
-                .push(text::bold(text::small("+ ")))
-                .push(text::bold(text::small(
-                    &ctx.converter.converts(*securing_amount).to_string(),
-                )))
-                .push(text::small(&format!(" {},   ", ctx.converter.unit)))
-                .push(text::bold(text::small(&nb_securing_vaults.to_string())))
-                .push(text::small(" vaults"))
-                .align_items(Align::End),
+                .push(
+                    Tooltip::new(
+                        tooltip_icon().size(10),
+                        "Secured funds are funds controlled by stakeholders only",
```

Comment:
You don't need to repeat `funds` :)
"Secured funds are controlled by stakeholders only"

Reply:
:coffee: Thanks